values.yaml

```diff
@@ -122,6 +122,8 @@ global:
 
   enableHoustonInternalAuthorization: false
 
+  enablePerHostIngress: false
+
```

Comment:
I think this per-host ingress configuration should be the only way to do it. I don't think we should allow the old way of doing it. If we switch to this new method then we reduce the amount of code we have to maintain and test while still providing the same features, just in a different way that is compatible with more environments.

Is there some reason we should keep the old way as an option?

Reply:
Slack thread <https://astronomer.slack.com/archives/C02J8JT58CC/p1715623166453439>

We agreed to keep the old way in, behind a feature flag, and run it through a deprecation phase.

Reply:
We can default this to true @pgvishnuram